cmake/ports/sranipal/portfile.cmake

```diff
@@ -0,0 +1,20 @@
+include(vcpkg_common_functions)
+set(SRANIPAL_VERSION 1.1.0.1)
+set(MASTER_COPY_SOURCE_PATH ${CURRENT_BUILDTREES_DIR}/src)
+
+if (WIN32)
+    vcpkg_download_distfile(
+        SRANIPAL_SOURCE_ARCHIVE
+        URLS https://hifi-public.s3.amazonaws.com/seth/sranipal-1.1.0.1-windows.zip
```

Comment:
should this URL point to somewhere not in the hifi s3 bucket?

Reply:
@daleglass could you upload this maybe?

Reply:
It was already here, thanks dale! You can change it to https://athena-public.s3.amazonaws.com/seth/sranipal-1.1.0.1-windows.zip @sethalves